packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -263,6 +263,11 @@ class ThemeData with Diagnosticable {
   /// a component theme parameter like [sliderTheme], [toggleButtonsTheme],
   /// or [bottomNavigationBarTheme].
   ///
+  /// In Material Design 3, the [IconButton.visualDensity] isn't affected by
```

Comment:
I think this comment probably belongs with the visualDensity API doc.

Reply:
Moved. Thanks!